PayButton/paybutton-server
# [Issue #151] feat: using real prices in the dashboard

Description:
Now we  use the price at the time of each XEC and BCH tx to calculate the dashboard data. Solves #151 

Test plan:
1. Log in as dev2@paybutton.org,
2. Create a paybutton for some XEC/BCH address that you can more easily check the last few payments value using the price at the moment,
3. Log out and back in (as dev2@paybutton.org) to sync the transactions for that address,
4. The dashboard information should reflect the expected values.


Remarks:
The following is also being fixed:
- The pipeline for generating the dashboard payment data was weirdly modifying the transaction objects so that `transaction.amount` holds the value instead of the amount transactioned.
- Upserting many price rows asynchronously was causing a race condition in the database.

Depends on:
- [x] #295
